transit_to_gif.py

```diff
@@ -122,7 +122,7 @@
     points = [int(i) for i in points if i]
     ways_all_nodes.extend(points)
     w["nodes_ref"] = points
-print("Chargement des ways terminé : {:d}".format(len(ways)))
+print("Chargement des ways termine : {:d}".format(len(ways)))
```

Comment:
why did you remove the accents in the prints ?

Reply:
I couldn't solve an ascii error conversion when using `print`: 
UnicodeEncodeError: 'ascii' codec can't encode character '\xe9' in position 27: ordinal not in range(128)
(checked it again this morning)

Reply:
thanks a lot, it works like a charm :+1: